documentation-src/metalsmith/stylesheets/components/_icons.scss

```diff
@@ -24,7 +24,7 @@
   }
 
   &-algolia-small {
-    background: url($icon-algolia-small)no-repeat center center / contain
```

Comment:
What happened to the variable? If it isn't use anymore, maybe it should be removed too?